www/index.html

```diff
@@ -189,6 +189,7 @@
                 </div>
 
                 <div id="configuration" style="display: none;" class="category-padding">
+                <div id="returntoArticle_top" style="text-align: center;"></div>
```

Comment:
I agree. This was more a quick solution attempt, especially given the discoverability issues on mobile (home button is hidden inside a clunky hamburger menu, taking more clicks to open and navigate around). Hence attempt (not very elegant, I agree!) to provide clear visual clues.